superset/assets/javascripts/chart/Chart.jsx

```diff
@@ -87,7 +87,7 @@ class Chart extends React.PureComponent {
   componentDidUpdate(prevProps) {
     if (
         this.props.queryResponse &&
-        this.props.chartStatus === 'success' &&
+        ['success', 'rendered'].indexOf(this.props.chartStatus) > -1 &&
```

Comment:
do you think we should make these `CONSTANTS` to make this less error-prone and ensure consistency between components?